packages/app-project/src/screens/SubjectTalkPage/components/SubjectTalkData/components/Tags/Tags.jsx

```diff
@@ -0,0 +1,204 @@
+import { Loader, SpacedText } from '@zooniverse/react-components'
+import { Box, Button, Heading } from 'grommet'
+import { Add, Tag as TagIcon } from 'grommet-icons'
+import { useTranslation } from 'next-i18next'
+import { arrayOf, bool, func, number, shape, string } from 'prop-types'
+import styled from 'styled-components'
+
+import Tag from './components/Tag'
+
+const StyledHeading = styled(Heading)`
+  letter-spacing: 0.8px;
+  text-transform: uppercase;
+`
+
+const StyledOrderedList = styled(Box)`
+  list-style: none;
```

Comment:
```suggestion
  list-style: none;
  row-gap: 10px;
  column-gap: 10px;
```

You can use [row-gap](https://developer.mozilla.org/en-US/docs/Web/CSS/row-gap) and [column-gap](https://developer.mozilla.org/en-US/docs/Web/CSS/column-gap) in a flex-wrapped div to apply spacing between its children. This is preferable to giving each child a right and bottom margin, because that applies extra margin to the right-most and bottom-row children, making this section taller than it appears in Figma.